openquakeplatform/openquakeplatform/vulnerability/views.py

```diff
@@ -748,7 +748,12 @@ def show_general_information(request, gen_info_id, action=None):
              edit_enabled=edit_enabled,
              curve_in_cart=curve_in_cart,
              cart_curves=cart_curves,
-             slug_name=slug_name),
+             slug_name=slug_name,
+             user_mailaddr="%s %s <%s>" % (
+                    gi.owner.first_name, gi.owner.last_name,
+                    gi.owner.email),
+             vuln_mailaddr="GEM Vulnerability Team <vulnerability@globalquakemodel.org>",
```

Comment:
Please be careful: this address does not exist (yet). Please check with @CatalinaYepes